.github/workflows/openapi.yaml

```diff
@@ -24,9 +29,14 @@ jobs:
         run: |
           rm ./trustify-ui/client/openapi/trustd.yaml
           cp ./trustify/openapi.yaml ./trustify-ui/client/openapi/trustd.yaml
-          
           cd ./trustify-ui
           git diff
+
+          if ! git diff --quiet; then
```

Comment:
This current GH action will only be triggered when a change to the file openapi.yaml is done, therefore this verification doesn't hurt but seems redundant. The `if` condition will always be `true`

https://github.com/trustification/trustify/blob/bc8a88defd2666bd6e6551be29ed215627ffc365/.github/workflows/openapi.yaml#L3-L8

Reply:
Ah.. I had not noticed that.. I'll simplify.